test/packages/good/manifest.yml

```diff
@@ -40,3 +40,4 @@ icons:
     title: system
     size: 1000x1000
     type: image/svg+xml
+elasticsearch.cluster: [monitor/main]
```

Comment:
My understanding is that this is only about cluster privileges and not cluster settings in general. I wonder if we should prefix it.

Reply:
Looking at your initial proposal and what have here, it seems we should prefix it: https://github.com/elastic/package-spec/pull/211

Reply:
something like `elasticsearch.cluster.privileges`?